bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/UserTarget.java

```diff
@@ -0,0 +1,70 @@
+package edu.gemini.spModel.target.env;
+
+import edu.gemini.shared.util.immutable.ImOption;
+import edu.gemini.shared.util.immutable.Option;
+import edu.gemini.spModel.target.SPTarget;
+
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+/**
+ * Tuple of a user SPTarget and its type.
+ */
+public final class UserTarget implements Serializable {
```

Comment:
oh, I thought we did set the version on the other classes. `Serialization` scares me